README.md

```diff
@@ -181,11 +180,11 @@ This module will use SemVer, and will stay on v0.X for the foreseeable future
 
 | Name | Description |
 |------|-------------|
-| cluster\_kubeconfig | KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
+| cluster\_kubeconfig | [SENSITIVE] KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)